qemu/tests/qemu_guest_agent.py

```diff
@@ -1512,6 +1525,21 @@ def gagent_check_file_read(self, test, params, env):
         :param params: Dictionary with the test parameters
         :param env: Dictionary with test environment.
         """
+        def _read_guest_file_with_count(count_num):
+            """
+            Read a guest file with count number.
+
+            :params: count_num: read file with count number on demand.
+            """
+            try:
+                self.gagent.guest_file_read(ret_handle, count=int(count_num))
+            except guest_agent.VAgentCmdError as detail:
+                info_insuffi = "Insufficient system resources exist to"
+                info_insuffi += " complete the requested service"
+                if info_insuffi not in str(detail.edata['desc']):
```

Comment:
Should the original type of `detail.edata['desc']` be string? I guess no need to convert here but I am not quite sure, you might check.

Reply:
okay, got it .